book/src/continuous_integration/github_actions.md

```diff
@@ -0,0 +1,22 @@
+# GitHub Actions
+
+## actions-rs
+
+An easy way to get started with adding Clippy to GitHub Actions is
+the [actions-rs](https://github.com/actions-rs) [clippy-check](https://github.com/actions-rs/clippy-check):
+
+```yml
+on: push
+name: Clippy check
+jobs:
+    clippy_check:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v1
+            -   run: rustup component add clippy
+            -   uses: actions-rs/clippy-check@v1
+                with:
+                    token: ${{ secrets.GITHUB_TOKEN }}
+                    args: --all-features
```

Comment:
This is not necessary. The latest stable Rust toolchain is already installed including Clippy on all GitHub provided runners. So just 
```suggestion
            -   name: Run Clippy
                run: cargo clippy
```

already does the job.

Reply:
For the different flags, I would just copy it like it is documented in the Travis section.